app/src/main/java/ch/epfl/sweng/favors/database/internal_db/InternalSqliteDb.java

```diff
@@ -0,0 +1,31 @@
+package ch.epfl.sweng.favors.database.internal_db;
+
+import android.content.Context;
+import android.database.sqlite.SQLiteDatabase;
+
+import ch.epfl.sweng.favors.database.User;
+
+import static android.database.sqlite.SQLiteDatabase.openOrCreateDatabase;
+
+public final class InternalSqliteDb {
+    private static SQLiteDatabase db;
+
+    public static String INTERNAL_DATABASE = "local_db";
```

Comment:
can this be final?

Reply:
yes for sure. thanks for pointing that out !